packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 9.1.5
+
+-   feat(suite): T2B1 replace Model R name by official Trezor Safe 3 name (7460372ed1)
+-   feat(connect): check that CA certificate validity from is not in the future (249ddc358a)
+-   feat(connect): add production root and CA pubkeys (af907a296d)
+-   feat(connect): add authenticateDevice method (45b99c0813)
+-   feat(connect): btg, dash, dgb, nmc, vrc no support for T2B1 (0819ff6fc1)
+-   feat(suite): support t2b1 firmware installation (9ef2bf627a)
+-   fix(connect): get firmware status and release after ensuring internal model (dca3333c2d)
+-   fix(connect-popup): webusb in popup if iframe is on same origin as host (e571971586)
+-   feat(connect-web): trust-issues query string param (b1b6e3f287)
+-   feat(connect-plugin-stellar): Update stellar-sdk and stellar-base (ee7e67db04)
```

Comment:
side note: we should release also this.

Reply:
ok, so then we need to update CHANGELOG for that `connect-plugin-stellar` package. Right?